_posts/2023-06-05-releases-2.4.x-2.5.0.md

```diff
@@ -0,0 +1,175 @@
+   this will default to a `1.0` value. Note, this does not affect the
+   topology, i.e. `AtomGroup.tempfactors` is not dynamically updated.
+   (Issue #3825, PR #3988)
+ - Add writing u.trajectory.ts.data['molecule_tag'] as molecule tags to
+   LAMMPS data file (Issue #3548)
+ - Add `progressbar_kwargs` parameter to `AnalysisBase.run` method, allowing
+   to modify description, position etc of tqdm progressbars. (PR #4085)
+ - Add a nojump transformation, which unwraps trajectories so that particle
+   paths are continuous. (Issue #3703, PR #4031)
+ - Added AtomGroup TopologyAttr to calculate gyration moments (Issue #3904,
+   PR #3905)
+ - Add support for TPR files produced by Gromacs 2023 (Issue #4047)
+ - Add distopia distance calculation library bindings as a selectable backend
+   for `calc_bonds` in `MDA.lib.distances`. (Issue #3783, PR #3914)
+ - AuxReaders are now pickle-able and copy-able (Issue #1785, PR #3887)
+ - Add pickling support for Atom, Residue, Segment, ResidueGroup 
+   and SegmentGroup. (PR #3953)
+
+### Changes:
+ - As per NEP29 the minimum supported Python version has been raised to 3.9
+   and NumPy has been raised to 1.21 (note: in practice later versions of NumPy
+   may be required depending on  your architecture, operating system, or Python
+   version) (PRs #4115 and #3983).
+ - Add progress bars to track the progress of `mds.EinsteinMSD` `_conclude()`
+   methods (`_conclude_simple()` and `_conclude_fft()`) (Issue #4070, PR #4072)
+ - The deprecated direct indexing and `times` from the `results` attribute of
+   analysis.nucleicacids' NucPairDist and WatsonCrickDist classes has been
+    removed. Please use the `results.pair_distances` and `times` attributes
+    instead (Issue #3744)
+ - RDKitConverter changes (part of Issue #3996):
+   - moved some variables (`MONATOMIC_CATION_CHARGES `and
+     `STANDARDIZATION_REACTIONS`) out of the related functions to allow users
+     fine tuning them if necessary.
+   - changed the sorting of heavy atoms when inferring bond orders and
+     charges: previously only based on the number of unpaired electrons, now
+     based on this and the number of heavy atom neighbors.
+   - use RDKit's `RunReactantInPlace` for the standardization reactions, which
+     should result in a significant speed improvement as we don't need to use
+     bespoke code to transfer atomic properties from the non-standardized mol
+     to the standardized one.
+
+## Author statistics
+
+This release was the work of 25 contributors, **15** of which are **new contributors.**
+
+Our **new contributors** are:
+
+@mglagolev
+@chrispfae
+@ooprathamm
+@MeetB7
+@v-parmar
+@MoSchaeffler
+@xhgchen
+@DrDomenicoMarson
+@AHMED-salah00
+@schlaicha
+@jvermaas
+@SophiaRuan
+@marinegor
+@g2707
+@DanielJamesEvans
+
+
+## MDAnalysis 2.4.x 
+
+MDAnalysis 2.4.0 and three follow up bug fix releases were made earlier this year.
+We unfortunately did not release a blog post for this release.
```

Comment:
No need to apologise for a missing blog post?

Reply:
I agree with @RMeli . It's good enough to mention it here, we've done that in the past. (Yes, one blog per release would be good but sometimes life happens.)

Reply:
```suggestion
```